packages/ui/src/domainModules/agent/components/AgentIdTextResult/index.js

```diff
@@ -106,14 +114,18 @@ class AgentIdTextResult extends Component {
     })
 
     if (textOnly) {
-      return <React.Fragment>{`${agentName} [${suffix}]`}</React.Fragment>
+      return (
+        <React.Fragment>
+          {`${agentName} ${disambiguatingDescriptionText} [${suffix}]`}
+        </React.Fragment>
+      )
     }
 
     return (
       <FieldTemplate {...extractedProps} name={name}>
         <div style={{ position: 'relative' }}>
           <strong>{agentName}</strong>
-          {` [${suffix}]`}
+          {` ${disambiguatingDescriptionText} [${suffix}]`}
```

Comment:
there's an unnecessary extra space in the beginning and it will lead to a double space if empty